package.json

```diff
@@ -86,5 +86,8 @@
     "packages": [
       "packages/*"
     ]
+  },
+  "dependencies": {
+    "@octokit/core": "^2.4.2"
```

Comment:
@molant any idea why we have to separately include this now? 😢

Also, shouldn't this still be under `devDependencies` like `@octokit/plugin-throttling`?

Reply:
> shouldn't this still be under devDependencies like @octokit/plugin-throttling?

Yes 🤦‍♂️

> any idea why we have to separately include this now?

No idea, guess they went through a refactor and they come up with a new `core` package? 🤷‍♂️

Reply:
This should be fixed now